app/animal-management-naive-dont-copy/read/[animalId]/page.js

```diff
@@ -2,8 +2,6 @@ import Image from 'next/image';
 import { notFound } from 'next/navigation';
 import { getAnimalById } from '../../../../database/animals';
 
-export const dynamic = 'force-dynamic';
```

Comment:
there are other `const dynamic = 'force-dynamic';` lines which need to be removed now too (after the PR #4 by David)

Reply:
https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/pull/3/commits/481dba24952155fd8af4218cb2a79d2c98f73b78
https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/pull/3/commits/1e962dbce8545972a9eadf9aed457cba79b17341